apps/root/src/pages/home/components/activity/index.tsx

```diff
@@ -173,14 +174,13 @@ const ActivityContent: ItemContent<TransactionEvent, Context> = (
         <StyledBodySmallRegularTypo2 noWrap={false}>
           {showBalances ? txTokenFlow : <HiddenNumber size="small" />}
         </StyledBodySmallRegularTypo2>
-        {status === TransactionStatus.PENDING ? (
-          <Chip
-            size="small"
-            variant="outlined"
-            color="primary"
-            sx={{ height: 'auto', '.MuiChip-label': { whiteSpace: 'normal', textAlign: 'center' } }}
-            label={<FormattedMessage defaultMessage="Waiting on confirmation" description="waiting-on-confirmation" />}
-          />
+        {status !== TransactionStatus.PENDING ? (
```

Comment:
this should be status === pending right?

Reply:
Ohhhh crap